shunting-yard-replicator/src/main/java/com/expediagroup/shuntingyard/replicator/exec/messaging/MessageReaderAdapter.java

```diff
@@ -138,4 +147,18 @@ private MetaStoreEvent map(ListenerEvent listenerEvent) {
     return builder.build();
   }
 
+  private boolean isPartitionedTable(String dbName, String tableName) {
+    Table sourceTable = null;
+    try {
+      sourceTable = sourceMetastoreClient.getTable(dbName, tableName);
+    } catch (TException e) {
+      throw new ShuntingYardException(String.format("Could not find table {}.{}", dbName, tableName), e);
+    }
+
+    if (CollectionUtils.isEmpty(sourceTable.getPartitionKeys())) {
```

Comment:
Also not a big deal but could be:

`return !CollectionUtils.isEmpty(sourceTable.getPartitionKeys())` right after line 153 in the try block.

Reply:
Yes. Patrick also mentioned this. I have changed to `isNotEmpty(..)`